sources/osg/Shape.js

```diff
@@ -330,9 +330,9 @@ define( [
         var vertexes = new MACROUTILS.Float32Array( [ -1.0, -1.0, -1.0, 4.0, 4.0, -1.0 ] );
 
         var indexes = new MACROUTILS.Uint16Array( 3 );
-        indexes[ 0 ] = 0;
+        indexes[ 0 ] = 2;
```

Comment:
when I added cull backface on composer... I saw the fullscreen triangle was inverted... so inverted it and added the cull backface on the composer stateset